inception-external-search-elastic/src/main/java/de/tudarmstadt/ukp/inception/externalsearch/elastic/traits/ElasticSearchProviderTraitsEditor.java

```diff
@@ -83,6 +83,13 @@ protected void onSubmit()
         TextField<String> objectType = new TextField<>("objectType");
         objectType.setRequired(true);
         form.add(objectType);
+    
+        NumberTextField<Integer> resultSize =
+                new NumberTextField<Integer>("resultSize", Integer.class);
+        resultSize.setMinimum(1);
+        resultSize.setMaximum(25000);
```

Comment:
Isn't 25000 a bit too much? Let's maybe start with 10000 and see if people complain.

Reply:
Changed in newest commit.